src/core/classPaths.ml

```diff
@@ -11,16 +11,25 @@ let create_resolved_file file class_path = {
 	class_path;
 }
 
+type file_resolution_specificity =
+	| Specificity_Normal            (* Standard Module.hx file *)
```

Comment:
I'm kind of moving away from these custom prefixes for variant constructors, but if we want to use them can we at least not go for camel... snake... case or whatever this is supposed to be? :P